src/.vuepress/util.js

```diff
@@ -148,10 +148,10 @@ function sortPagesByOrderAndTitle(p1, p2) {
   const o1 = get(p1, 'frontmatter.order', null);
   const o2 = get(p2, 'frontmatter.order', null);
 
-  if ((o1 === null || typeof o1 === 'undefined') && o2) {
+  if ((o1 === null || typeof o1 === 'undefined') && !isNaN(o2)) {
```

Comment:
if o1 or o2 is NaN, then that function returns 1 (because any comparison operation on a NaN value ends up returning `false`)

also, I don't get the purpose of that test, what does it do?

Reply:
Both `if` serve to check if `o1` and `o2` are valids. I added `!isNaN()` because without that, if `o1` or `o2` equals to zero, the sorting is broken

Reply:
0 !== NaN (everything !== NaN)

I'm not sure that this test does what you want it to do

Reply:
I don't get it, because NaN is not 0

Reply:
When `o2` is equal to zero, the condition consider that `o2` is not a valid number. This produce broken sorting of pages because a page with `order` set to zero wasn't the first page in the list